docs/graphviz.rst

```diff
@@ -0,0 +1,131 @@
+Graphviz
+========
+
+This theme adds several improvements to the built-in :py:mod:`sphinx.ext.graphviz` extension:
+
+- default fonts and colors are set to match the fonts and colors used by the
+  theme, including support for both light and dark mode;
+
+- the rendered diagram is included as inline svg content to allow hyperlinks to
+  work;
+
+- colors can be manually specified in terms of CSS variables defined by the
+  theme, such as :css:`var(--md-primary-fg-color)`;
+
+- labels can be specified as Sphinx cross references (rather than just plain
+  URLs).
+
+Configuration
+-------------
+
+This functionality is available as a separate optional extension included with
+this theme, which must be specified manually in your :file:`conf.py` file:
+
+.. code-block:: python
+
+    extensions = [
+        "sphinx_immaterial",
+        # other extensions...
+        "sphinx_immaterial.graphviz",
+    ]
+
+The :confval:`graphviz_dot` and :confval:`graphviz_dot_args` configuration
+options from :py:mod:`sphinx.ext.graphviz` are supported.
+
+.. note::
+
+   The :confval:`graphviz_output_format` configuration option is not supported;
+   instead, the graph is always included as inline SVG in the HTML output.
+
+.. confval:: graphviz_ignore_incorrect_font_metrics
+
+   This extension relies on the LibGD graphviz plugin to load the same font used
+   by this theme, in order to correctly determine the size of text labels.
+   While the LibGD plugin is normally included in the Linux and macOS graphviz
+   builds, the official x86_64 Windows build does not include it.  If the plugin
+   is not found, by default this theme logs a warning.  This option may be set
+   to :python:`True` silence that warning.
+
+   .. rst-example:: Add to :file:`conf.py` to silence the warning
+
+      graphviz_ignore_incorrect_font_metrics = True
```

Comment:
I missed this. Probably didn't mean to use rst-example
![image](https://user-images.githubusercontent.com/14963867/187320474-d5a31bbd-a65c-467c-9de2-d4cf11ac217c.png)